tools/ci_connector_ops/ci_connector_ops/pipelines/commands/groups/metadata.py

```diff
@@ -37,7 +42,7 @@ def metadata(ctx: click.Context):
 @click.pass_context
 def validate(ctx: click.Context, modified_only: bool) -> bool:
     if modified_only:
-        metadata_to_validate = get_modified_metadata_files(ctx.obj["modified_files"])
+        metadata_to_validate = get_expected_metadata_files()
     else:
         click.secho("Will run metadata validation on all the metadata files found in the repo.")
```

Comment:
Is is the `run_metadata_validation_pipeline` that will do the yelling if the expected metadata file is not found? (either because it's missing or incorrectly named)

Reply:
In this case the pipeline will fail
https://app.warp.dev/block/z0MDguT3n3kn1G6A2nPno2

This is because the pipeline goes
1. Oh, a connector has changed
2. A metadata file must exist at this path

Then when it goes to add the file that doesnt exist it fails